manager/controllers/package_detail_test.go

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import (
+	"app/base/core"
+	"app/base/utils"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestLatestPackage(t *testing.T) {
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "/packages/kernel", nil)
+	core.InitRouterWithParams(PackageDetailHandler, "3", "GET", "/packages/:package_name").
+		ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+	var output PackageDetailResponse
+	ParseReponseBody(t, w.Body.Bytes(), &output)
+	assert.Equal(t, "kernel", output.Data.Attributes.Name)
+	assert.Equal(t, "RH-7", output.Data.Attributes.AdvisoryID)
+	assert.Equal(t, "5.6.13-201.fc31.x86_64", output.Data.Attributes.Version)
```

Comment:
Please rather check all result fields (desc., summ, ID), thanks. Also for ExactPackage test.